pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -306,7 +307,13 @@ func buildNetworkTopologyMetadata(deviceID string, store *metadata.Store, interf
 
 		localInterfaceIDType, localInterfaceID = resolveLocalInterface(deviceID, interfaceIndexByIDType, localInterfaceIDType, localInterfaceID)
 
+		// remEntryUniqueID: The combination of localPortNum and lldpRemIndex is expected to be unique for each entry in
+		//                   lldpRemTable. We don't include lldpRemTimeMark (used for filtering only recent data) since it's can change often.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
```suggestion
		//                   lldpRemTable. We don't include lldpRemTimeMark (used for filtering only recent data) since it can change often.
```